clang/lib/Sema/SemaType.cpp

```diff
@@ -2123,7 +2123,7 @@ QualType Sema::BuildPointerType(QualType T,
 
   // If we are in purecap ABI turn pointers marked as using an integer
   // representation into a plain pointer-range-sized integer
-  if (PointerInterpretation == PIK_Integer
```

Comment:
I got a bit confused by this diff initially. That seems to indicate that I should not have named the Sema member `PointerInterpretation`...

Reply:
Yeah I was initially confused where that was coming from. That's David's field though (6232a3eb6e7db), so you're off the hook :) though we have both touched it over the years.